package.json

```diff
@@ -229,7 +229,7 @@
     "unist-util-visit": "^2.0.3",
     "uuid": "^8.3.2",
     "vast-client": "^3.1.1",
-    "video.js": "^7.13.3",
+    "video.js": "git://github.com/mayeaux/video-js-fork",
     "videojs-contrib-quality-levels": "^2.0.9",
     "videojs-event-tracking": "^1.0.1",
```

Comment:
Press "pause", wait a while, and "play".  The "LIVE" button no longer works (keeps going back to dimmed).

<img width="175" alt="image" src="https://user-images.githubusercontent.com/64950861/155057312-9acd38be-3347-4b20-98d9-bd89da4487af.png">

Reply:
If you playing at x0.25 speed, we are no longer live, but it continues to be lit.